packages/vuetify/src/styles/elements/_typography.sass

```diff
@@ -1,88 +1,84 @@
-.v-application
-  .display-4
-    font-size: map-deep-get($headings, 'h1', 'size') !important
-    font-weight: map-deep-get($headings, 'h1', 'weight')
-    line-height: map-deep-get($headings, 'h1', 'line-height')
-    letter-spacing: map-deep-get($headings, 'h1', 'letter-spacing') !important
-    font-family: map-deep-get($headings, 'h1', 'font-family') !important
+.display-4
```

Comment:
this entire file can actually go away, as well as the loop in `settings/variables.scss` to make the new typography names backwards compatible.